### PR TITLE
example/wifi_manager: Add a stress test for wifi_manager

### PR DESCRIPTION
**Pull request by pillip8282**
added features
1) check elapsed time of a function
2) repeat tests and check successes and fails

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [a21b4cd38efed283e06ba1898a8ff56f7a4f1830] - Code Rule Check Result:
apps/examples/wifi_manager_sample/rt_perf.c:108: ERROR: [IDT_M_TAB] code indent should use tabs where possible
apps/examples/wifi_manager_sample/rt_perf.c:108: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
apps/examples/wifi_manager_sample/rt_perf.c:112: ERROR: [IDT_M_TAB] code indent should use tabs where possible
apps/examples/wifi_manager_sample/rt_perf.c:112: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
apps/examples/wifi_manager_sample/rt_perf.h:95: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
apps/examples/wifi_manager_sample/rt_perf.h:100: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
apps/examples/wifi_manager_sample/rt_perf.h:154: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
apps/examples/wifi_manager_sample/rt_perf.h:167: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
apps/examples/wifi_manager_sample/rt_perf.h:178: ERROR: [SPC_M_SEP] space required after that close brace '}'
apps/examples/wifi_manager_sample/rt_perf.h:180: ERROR: [SPC_M_OPR] space prohibited after that '*' (ctx:BxW)
apps/examples/wifi_manager_sample/rt_perf.h:235: ERROR: [SPC_M_KWD] missing space after struct definition
apps/examples/wifi_manager_sample/wm_test_stress.c:155: ERROR: [SPC_M_OPR] space required after that ',' (ctx:VxV)

**Comment by seinfra** [comment]
Target : [a21b4cd38efed283e06ba1898a8ff56f7a4f1830] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [a21b4cd38efed283e06ba1898a8ff56f7a4f1830] -

**Comment by seinfra** [comment]
Target : [97dcbf1068c88bebea2d4297da279b1d4781c23a] -

**Comment by seinfra** [comment]
Target : [97dcbf1068c88bebea2d4297da279b1d4781c23a] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [97dcbf1068c88bebea2d4297da279b1d4781c23a] - Code Rule Check (C++) OK.

File: apps/examples/wifi_manager_sample/Kconfig [hunk 1]
```diff
@@ -10,6 +10,78 @@ config EXAMPLES_WIFIMANAGER_TEST
 	---help---
 		Wi-Fi Manager sample program
 
+
 config USER_ENTRYPOINT
 	string
 	default "wm_test_main" if ENTRY_WIFIMANAGER_TEST
+
+
+if EXAMPLES_WIFIMANAGER_TEST
```
Comment (sangwon03) [inline apps/examples/wifi_manager_sample/Kconfig]: I think 'EXAMPLES_WIFIMANGER_STRESS_TEST' is more proper word than 'EXAMPLES_WIFIMANGER_TEST'

Comment (pillip8282) [reply apps/examples/wifi_manager_sample/Kconfig]: Ok I'll fix

**Pull request merged**